webpack/webpack.config.dev-client.js

```diff
@@ -84,11 +84,10 @@ module.exports = {
       publicPath: publicPath
     },
     module: {
-      loaders: commonLoaders.concat([
-        { test: /\.css$/,
-          loader: 'style!css?module&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
-        }
-      ])
+      loaders: commonLoaders.concat({
+        test: /\.css$/,
+        loader: 'style!css?module&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
```

Comment:
I found that official document says we should use **?modules** to use css-modules. But we have used **?module** without **s** till now.
[Link](https://github.com/webpack/css-loader#css-modules)